src/Sample.xoo

```diff
@@ -10,3 +10,6 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:4000/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:4000/project/issues?id=sample&pullRequest=5137&issues=AWV7cg9u1fJTOgITn8JK&open=AWV7cg9u1fJTOgITn8JK)

[ISSUE_KEY]: AWV7cg9u1fJTOgITn8JK